README.md

```diff
@@ -180,8 +180,10 @@ To run tests, run the following command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- Features -->
-### Key Features <a name="key-features"></a>
```

Comment:
Well done documenting the README. Everything looks great 👏🏾👍🏾

- Please add the `Key Features`  section you deleted and make sure that you have at least [1 - 3] features listed. Please never remove any headings and sub-headings on the template except for the FAQ section. It is the only one that is optional.

Reply:
Thanks @KDlamini ; resolution considered